tests/pytest/enrollment_switchio/test_api.py

```diff
@@ -3,21 +3,62 @@
 import pytest
 
 import benefits.enrollment_switchio.api
-from benefits.enrollment_switchio.api import Client, EshopResponseMode, Registration, RegistrationMode, RegistrationStatus
+from benefits.enrollment_switchio.api import (
+    EnrollmentClient,
+    Group,
+    TokenizationClient,
+    EshopResponseMode,
+    Registration,
+    RegistrationMode,
+    RegistrationStatus,
+)
 
 
 @pytest.fixture
-def client():
-    return Client("https://example.com", "api key", "api secret", None, None, None)
+def tokenization_client():
+    return TokenizationClient("https://example.com", "api key", "api secret", None, None, None)
+
+
+@pytest.fixture
+def enrollment_client():
+    return EnrollmentClient("https://example.com", "Basic abc123", None, None, None)
+
+
+@pytest.mark.django_db
+def test_tokenization_client_cert_request(mocker, tokenization_client):
+    temp_file = mocker.patch("benefits.enrollment_switchio.api.NamedTemporaryFile")
+    request_func = mocker.Mock()
+
+    tokenization_client._cert_request(request_func)
+
+    temp_file.assert_called()
+    request_func.assert_called_once()
+    assert "verify" in request_func.call_args.kwargs
+    assert "cert" in request_func.call_args.kwargs
+
+
+@pytest.mark.django_db
+def test_enrollment_client_cert_request(mocker, enrollment_client):
+    temp_file = mocker.patch("benefits.enrollment_switchio.api.NamedTemporaryFile")
+    request_func = mocker.Mock()
+
+    enrollment_client._cert_request(request_func)
```

Comment:
Why not test `_cert_request` on the parent `Client` directly, rather than on both child classes?